packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -41,7 +42,13 @@ class DevToolsAppState extends State<DevToolsApp> {
     final path = uri.path;
 
     if (_routes.containsKey(path)) {
-      return MaterialPageRoute(settings: settings, builder: _routes[path]);
+      var builder = _routes[path];
+      assert(() {
+        builder =
+            (context) => _AlternateCheckedModeBanner(builder: _routes[path]);
```

Comment:
we don't need the checked mode banner. Go ahead and just hide it by default instead.

Reply:
+1

Reply:
It's more than an affordance for understanding that checked mode is slow.  This banner is also a warning that a release has accidentally been built with the wrong mode of the app.  This has saved several teams' release engineers a couple of times when doing releases, even inside Google.

Especially if we look into shipping desktop app builds along with the web app, having the little extra safety will be very helpful.

Reply:
Ok lets leave it as is. Coming from a web background it seems unnecessary but maybe it is more important on desktop/mobile where you can't quickly view source to see whether you shipped the wrong files.